ui/components/atoms/A009-Input/stories.js

```diff
@@ -9,11 +9,11 @@ import notes from './readme.md'
 // List of stories of use cases for atom
 storiesOf('Atoms|A009 - Input', module)
   .addDecorator(withNotes)
-  .add('Input', () =>
+  .add('Input : Without label', () =>
     <TextField id='1' type='text' name='name' placeholder='placeholder' />,
   {notes: { markdown: notes }
   })
-  .add('Input with label', () =>
+  .add('Input : With label', () =>
```

Comment:
this actually shouldn't be here - this atom isn't meant to have a label. anyway we obviously missed it.